my_store/blog/dashboard/apps.py

```diff
@@ -2,8 +2,7 @@
 
```

Comment:
I think you should change to use _ to - in url-name such as blog_list_view => blog-list-view